scripts/mutect2_wdl/unsupported/mutect2_opt.wdl

```diff
@@ -58,10 +54,25 @@ workflow Mutect2 {
   String? sequencing_center
   String? sequence_source
   File? default_config_file
+  File? tumor_sequencing_artifact_metrics
   Boolean is_bamOut = false
 
   # Do not populate unless you know what you are doing...
   File? auth
+  # This value is added to every tasks disk in case the dynamic sizing isnt enough
+  Int? emergency_extra_disk
+
+  # Disk sizes used for dynamic sizing
+  Int ref_size = ceil(size(ref_fasta, "GB") + size(ref_dict, "GB") + size(ref_fasta_index, "GB"))
+  Int tumor_bam_size = ceil(size(tumor_bam, "GB") + size(tumor_bam_index, "GB"))
+  Int gnomad_vcf_size = if defined(gnomad) then ceil(size(gnomad, "GB") + size(gnomad_index, "GB")) else 0
+  Int normal_bam_size = if defined(normal_bam) then ceil(size(normal_bam, "GB") + size(normal_bam_index, "GB")) else 0
+  Int onco_tar_size = if defined(onco_ds_tar_gz) then ceil(size(onco_ds_tar_gz, "GB") * 3) else 100
```

Comment:
This is actually one of the things I had no idea what it was doing.  Because the tar get unzipped in the task I dont really know what mulitplier I should be using here.  100 was the default value in the original wdl and just what Lee said they used.  That's really the only reason.

Reply:
oh if you dont define a tar then it goes to an ftp server and downloads a tar for you, thus why the `else` part is not 0

Reply:
ah got it, actually that might be worth a comment (that if you don't provide a tar it downloads it for you)

Reply:
👍